components/dialog/dialog-mixin.js

```diff
@@ -17,6 +17,7 @@ if (window.D2L.DialogMixin.preferNative === undefined) {
 	window.D2L.DialogMixin.preferNative = true;
 }
 
+const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
```

Comment:
This means you have to reload the page after changing your OS preference, ya? Not a big deal and likely worth the optimization trade-off.

Reply:
That's correct.  

If we want... we could run this each time it's needed, or we could be notified when it changes, which is useful for other types of media queries where the match result is expected to change.  Unwrapped, it would look something like:
```
const query = window.matchMedia('(prefers-reduced-motion: reduce)');
let reduceMotion = query.matches;
query.addListener((e) => reduceMotion = e.matches);
```
But because we wouldn't want lots of places defining that same code, we'd probably wrap it up in an object that observes that event, which would result in an extra import, which probably isn't worth it in this case as you mentioned.  :)

I think there might also be some trickiness with IE11, depending on whether `addListener` or `addEventListener` is used, but I'd have to retest it to see where it was falling over.

Reply:
Probably not worth it, mostly just curious if we'd get amazing instant results like with dark mode -- which would be sweet for demos and that's about all.